lte/gateway/docker/services/s1aptester/Dockerfile

```diff
@@ -0,0 +1,117 @@
+ARG PYSTEMD_VERSION=0.7.0-2build1
+ARG PYTHON_IS_PYTHON3_VERSION=3.8.2-4
+ARG RUBY_VERSION=1:2.7+1
+ARG RUBY_DEV_VERSION=1:2.7+1
+ARG RSYNC_VERSION=3.1.3-8ubuntu0.3
+ARG SSH_VERSION=1:8.2p1-4ubuntu0.5
+ARG SSHPASS_VERSION=1.06-1
+ARG SUDO_VERSION=1.8.31-1ubuntu1.2
+ARG VENV_VERSION=3.8.2-0ubuntu2
+ARG VIRTUALENV_VERSION=20.0.17-1ubuntu0.4
+ARG WGET_VERSION=1.20.3-1ubuntu2
+
+ENV VIRTUAL_ENV=/build/venv
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+ENV MAGMA_ROOT=/magma
+ENV S1AP_TESTER_ROOT=/s1ap-tester
+ENV S1AP_TESTER_SRC=/S1APTester
+ENV PYTHON_BUILD=/build
+ENV PIP_CACHE_HOME="~/.pipcache"
+ENV TZ=Europe/Paris
+ENV SWAGGER_CODEGEN_JAR=/var/tmp/codegen/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar
+ENV SWAGGER_CODEGEN_DIR=/var/tmp/codegen/
+ENV S1AP_TESTER_CFG=$MAGMA_ROOT/lte/gateway/python/integ_tests/data/s1ap_tester_cfg
+ENV GRPC_POLL_STRATEGY=poll
+
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends \
+  apt-utils=$APT_UTILS_VERSION \
+  binutils=$BINUTILS_VERSION \
+  build-essential=$BUILD_ESSENTIAL_VERSION \
+  git=$GIT_VERSION \
+  libiperf0=$LIBIPERF0_VERSION \
+  libmysqlclient-dev=$LIBMYSQL_VERSION \
+  libprotobuf-dev=$LIBPROTOBUF_VERSION \
+  libsystemd-dev=$LIBSYSTEMD_DEV_VERSION \
+  lsb-release=$LSB_RELEASE_VERSION \
+  openjdk-8-jdk=$JDK_VERSION \
+  pkg-config=$PKG_CONFIG_VERSION \
+  python-is-python3=$PYTHON_IS_PYTHON3_VERSION \
+  python3-eventlet=$EVENTLET_VERSION \
+  python3-pip=$PIP_VERSION \
+  python3-protobuf=$PROTOBUF_VERSION \
+  python3-pystemd=$PYSTEMD_VERSION \
+  python3-venv=$VENV_VERSION \
+  rsync=$RSYNC_VERSION \
+  ruby=$RUBY_VERSION \
+  ruby-dev=$RUBY_DEV_VERSION \
+  ssh=$SSH_VERSION \
+  sshpass=$SSHPASS_VERSION \
+  sudo=$SUDO_VERSION \
+  virtualenv=$VIRTUALENV_VERSION \
+  wget=$WGET_VERSION && \
+  apt-get clean && \
+  gem install fpm
+
+COPY . $MAGMA_ROOT/
+
+WORKDIR /var/tmp/
+
+RUN DIRECTORY="/tmp/aioeventlet" && \
+  mkdir -p ${DIRECTORY} && \
+  /magma/third_party/build/bin/aioeventlet_build.sh ${DIRECTORY} && \
+  dpkg -i ${DIRECTORY}/python3-aioeventlet* && \
+  rm -rf ${DIRECTORY} && \
+  mkdir -p ${SWAGGER_CODEGEN_DIR}modules/swagger-codegen-cli/target/; \
+  wget -q https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${CODEGEN_VERSION}/swagger-codegen-cli-${CODEGEN_VERSION}.jar -O ${SWAGGER_CODEGEN_JAR}
+
+WORKDIR $MAGMA_ROOT/lte/gateway/python
+
+RUN make && \
+  mkdir -p $S1AP_TESTER_ROOT/bin /var/tmp/test_results/
+
+WORKDIR $MAGMA_ROOT/lte/gateway/python/integ_tests
+
+RUN make && \
+  rm -rf /var/lib/apt/lists/* && \
+  $MAGMA_ROOT/lte/gateway/deploy/roles/magma_test/files/clone_s1_tester.sh && \
+  $MAGMA_ROOT/lte/gateway/deploy/roles/magma_test/files/build_s1_tester.sh && \
+  pip3 install --cache-dir $PIP_CACHE_HOME pyparsing && \
+  python3 $MAGMA_ROOT/lte/gateway/deploy/roles/magma_test/files/c_parser.py && \
+  sed -i 's/magtivate_cmd + " && " + //g' $MAGMA_ROOT/lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py && \
```

Comment:
As @Neudrino mentioned, this is fragile and should be fixed. And in this specific case it broke with a commit three months ago: https://github.com/magma/magma/commit/f70afcdc9dad27e6ca8122a703f43a669cf896e5#diff-6571bbc4f34804dcabb03702090a753fe91cdabb3b453d7228f4c09e2615f958R861